### DISCUSSION TITLE
How is fixture scope set?

### DISCUSSION THREAD

**Discussion by jasonkuhrt** [vitest-dev/vitest#5043] [Q&A] [answered]
Is there a way to set fixture scope? There was in the original library it seems but I can't find it in Vitest docs and trying it seems to not work.

Is a feature request needed?

https://github.com/larsthorup/vitest-fixture/blob/main/doc/api.md#fixture-scope

**Comment by sheremet-va** [comment, marked as answer]
We don't support this feature. PR or a feature request issue is welcome.
